apps/mobile/src/state/marketplace/atoms/offersToSeeInMarketplace.ts

```diff
@@ -53,21 +53,25 @@ export const offersToSeeInMarketplaceAtom = atom((get) => {
   return offers.filter(
     (oneOffer) =>
       // only active offers
-      oneOffer.offerInfo.publicPart.active &&
-      // only not expired offers
-      !isOfferExpired(oneOffer.offerInfo.publicPart.expirationDate) &&
-      // Not mine offers
-      !oneOffer.ownershipInfo &&
-      // Not reported offers
-      !oneOffer.flags.reported &&
-      // Offers that has at least one common contact or are first degree
-      (oneOffer.offerInfo.privatePart.commonFriends.some((one) =>
-        importedContactsHashes.includes(one)
-      ) ||
-        oneOffer.offerInfo.privatePart.friendLevel.includes('FIRST_DEGREE')) &&
-      // Filter offers that are set to be in person but have no location
-      (isDeveloper ||
-        oneOffer.offerInfo.publicPart.locationState.includes('IN_PERSON') ||
```

Comment:
I am not sure about this one, so correct me if I am wrong, but the old condition filtered out all `ONLINE` offers or? In new version I need also the `locationState.length = 0` to pass as there can be ONLINE offer without `locationState` set

Reply:
The condition before location state was an array was: It should return true if the offer is online or if the length of specified locations is 0